test/test_build_python.py

```diff
@@ -94,108 +147,44 @@ def _test_build_package(tmp_path_str, *, symlink_install):
         build_base = Path(python_build_task.context.args.build_base)
         assert build_base.rglob('my_module/__init__.py')
 
-        return Path(python_build_task.context.args.install_base)
```

Comment:
Since we no longer return anything from this function I went through the diff to make sure we've dropped all calls that save a return value and it appears we do.